core/MyGatewayTransportEthernet.cpp

```diff
@@ -120,36 +149,31 @@ bool gatewayTransportInit(void)
 #if defined(MY_ESP8266_SSID)
```

Comment:
For this section, it would be possible to consolidate the ESP8266 and ESP32 code. The only difference seems to be WiFi.hostname for ESP8266 vs Wifi.setHostname for ESP32.

Would it make sense to do
```
#if defined MY_GATEWAY_ESP8266
WiFi.hostname(MY_HOSTNAME)
#else if defined MY_GATEWAY_ESP32
WiFi.setHostname(MY_HOSTNAME)
#endif
```
and use the same code for everything else?